prow/cron/cron.go

```diff
@@ -0,0 +1,172 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package cron provides a wrapper of robfig/cron, which manages schedule cron jobs for horologium
+package cron
+
+import (
+	"fmt"
+	"strings"
+	"sync"
+
+	"github.com/robfig/cron"
+
+	"k8s.io/test-infra/prow/config"
+)
+
+// jobStatus is a cache layer for tracking existing cron jobs
+type jobStatus struct {
+	entryID cron.EntryID
+	trigger bool
```

Comment:
maybe doc this field? it's not immediately obvious without reading the rest what this field means